packages/forma-36-react-components/src/components/Notification/NotificationItem.test.tsx

```diff
@@ -13,6 +13,27 @@ it('renders the component', () => {
   expect(output).toMatchSnapshot();
 });
 
+it('renders the component with a title and a body', () => {
+  const output = shallow(
+    <NotificationItem title="Notification title" intent="success">
+      This is the body text.
+    </NotificationItem>,
+  );
+  expect(output).toMatchSnapshot();
```

Comment:
I thought we want to get rid of Snapshot tests. Or is this just valid for the `user_interface` codebase?

See e.g. [this article](https://blog.usejournal.com/whats-wrong-with-snapshot-tests-37fbe20dfe8e?gi=2ba83c92cb9) to get an idea why snapshot testing usually isn't a good idea.